sdk/ai/Azure.AI.Inference/src/Customized/ChatCompletionsClient.cs

```diff
@@ -47,7 +48,18 @@ public virtual async Task<Response<ChatCompletions>> CompleteAsync(ChatCompletio
 
             using RequestContent content = chatCompletionsOptions.ToRequestContent();
             RequestContext context = FromCancellationToken(cancellationToken);
-            Response response = await CompleteAsync(content, extraParams?.ToString(), context).ConfigureAwait(false);
+            using OpenTelemetryScope otel = new OpenTelemetryScope(chatCompletionsOptions, _endpoint);
+            Response response = null;
+            try
+            {
+                response = await CompleteAsync(content, extraParams?.ToString(), context).ConfigureAwait(false);
+                otel.RecordResponse(ChatCompletions.FromResponse(response));
+            }
+            catch (Exception ex)
```

Comment:
How does this work if a person doesn't care about telemetry? I don't want us throwing an exception if the user isn't even trying to use the feature

Reply:
In this case the behavior is not changed. We re-throw only if the exception was already thrown in the  Complete[Async]. If the telemetry is off otel.RecordResponse() is essentially no-op.